### PR TITLE
refactor(slider): improve code readability and adjust code style

### PR DESCRIPTION
**Pull request by wsuwt**
## Description

Improve code readability and adjust code style.

## Type of change
Please delete options that are not relevant.

- [x] Code refactoring

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/slider/index.ts [hunk 1]
```diff
@@ -263,104 +334,123 @@ export class Slider extends ControlElement {
    * Getter for slider part.
    */
   @query('[part="slider"]')
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/slider/index.ts]: All `query` decorator should be changed to `ref` decorator instead.

File: packages/elements/src/slider/index.ts [hunk 2]
```diff
@@ -88,12 +48,95 @@ export class Slider extends ControlElement {
...
+
+  /**
+   * Current active slider thumb
+   */
+  private activeThumb!: HTMLDivElement;
+
+  /**
+   * An array of slider thumbs
+   */
+  private thumbs: Array<HTMLDivElement> = [];
```
Comment (goremikins) [inline packages/elements/src/slider/index.ts]: private thumbs: HTMLDivElement[] = [];

File: packages/elements/src/slider/index.ts [hunk 3]
```diff
@@ -88,12 +48,95 @@ export class Slider extends ControlElement {
     return VERSION;
   }
 
-  private activeThumb!: HTMLElement;
+  /**
+   * Define styles in a tagged template literal, using the css tag function.
+   * @returns CSS template
+   */
+  static get styles (): CSSResultGroup {
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/slider/index.ts:56]: This method should be moved to the sibling `render` method. Ref: https://confluence.refinitiv.com/display/ELF/Code+Styling+Guide

Comment (goremikins) [reply packages/elements/src/slider/index.ts:56]: It should not. static methods come at the top. We need to change the guidelines

File: packages/elements/src/slider/utils.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+/**
+  * Set prevent default action and stop bubbles event
+  * @param event event mouse or touch
+  * @returns {void}
+  */
+const preventDefault = function (event: Event): Event {
```
Comment (goremikins) [inline packages/elements/src/slider/utils.ts]: Please use arrow functions:

`const preventDefault = (event: Event): Event => {`

**Comment by goremikins** [comment]
I think for light refactor it is good enough

File: packages/elements/src/slider/index.ts (hunk 1 above)
Comment (wsuwt) [reply packages/elements/src/slider/index.ts]: I changed all `@query` except inputs which are dynamically render and involve a lot of complexity and I think is beyond the current scope.

Comment (wsuwt) [reply packages/elements/src/slider/index.ts]: I'll save this for future refactoring.

File: packages/elements/src/slider/index.ts (hunk 3 above)
Comment (wsuwt) [reply packages/elements/src/slider/index.ts:56]: I agree that static methods should be on top.

**Review by goremikins — APPROVED** [review]

**Pull request merged**